lib/controller.js

```diff
@@ -239,17 +239,28 @@ Tessel.get = function(opts) {
         }
       }
 
+
+      function finishSearchEarly(tessel) {
+        // Remove this listener because we don't need to search for the Tessel
+        seeker.removeListener('end', searchComplete);
+        // Stop searching
+        seeker.stop();
+        // Send this Tessel back to the caller
+        logAndFinish(tessel);
```

Comment:
@rwaldron `seeker` should be passed as an argument to `logAndFinish`? The definition of `logAndFinish` looks like it just accepts one argument (a Tessel Object).